Visual Studio/Realm.vcxproj

```diff
@@ -221,7 +221,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Static library, release|x64'" Label="Configuration">
     <ConfigurationType>StaticLibrary</ConfigurationType>
-    <CharacterSet>Unicode</CharacterSet>
+    <CharacterSet>MultiByte</CharacterSet>

```

Comment:
This is going to be superseded by CMake so the project file doesn't really matter, as long as all the platform and configuration combos have the same value.